ocs_ci/ocs/ui/base_ui.py

```diff
@@ -106,6 +106,10 @@ def do_send_keys(self, locator, text, timeout=30):
             logger.error(e)
             raise TimeoutException
 
+    def do_click_and_send_keys(self, locator, text, timeout=30):
```

Comment:
Why do you need "click" and "send_keys" on same method?

Reply:
done